snapshot/lib/snapshot/collector.rb

```diff
@@ -97,7 +97,9 @@ def self.find_screenshot(activity)
         return find_screenshot(subactivity) if subactivity["Title"] == "Synthesize event"
       end
 
-      if activity["Attachments"]
+      if Helper.xcode_at_least?("9") && activity["Attachments"]
```

Comment:
Maybe we shouldn't actually check the version, but just see if the key is available? Not sure if that's better, both approaches are totally fine I think